src/go/k8s/pkg/resources/certmanager/keystore_password.go

```diff
@@ -0,0 +1,88 @@
+package certmanager
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	redpandav1alpha1 "github.com/vectorizedio/redpanda/src/go/k8s/apis/redpanda/v1alpha1"
+	"github.com/vectorizedio/redpanda/src/go/k8s/pkg/labels"
+	"github.com/vectorizedio/redpanda/src/go/k8s/pkg/resources"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	passwordKey = "password"
+
+	keystore = "keystore"
+)
+
+// KeystoreResource is part of the reconciliation of redpanda.vectorized.io CRD
+// creating secrets for pkcs#12 and jks password
+type KeystoreResource struct {
+	k8sclient.Client
+	scheme       *runtime.Scheme
+	pandaCluster *redpandav1alpha1.Cluster
+	key          types.NamespacedName
+	logger       logr.Logger
+}
+
+// NewKeystoreResource creates KeystoreResource instance
+func NewKeystoreResource(
```

Comment:
this is not Keystore resource, could you rename to `KeystoreSecretResource`? I was very confused by this in the beginning